src/components/AudioPlayer.tsx

```diff
@@ -54,8 +54,10 @@ const AudioPlayer = ({ document, disabled, submittedAlternative }: AudioPlayerPr
         }
       })
       .catch(async (error: TtsError) => {
+        /* eslint-disable-next-line no-console */
```

Comment:
is there a reason why not to use `reportError` fkt here?

Reply:
Wasn't sure this needed reporting. At least on my devices, this happens constantly. Apparently the original implementer also didn't see the value in logging the actual error, so I'm assuming this is only being logged to satisfy `typescript-eslint/no-floating-promises`